src/tsconfig.base.json

```diff
@@ -0,0 +1,41 @@
+{
+  "extends": "../tsconfig.json",
```

Comment:
This feels backwards. Can we reverse the dep and have tsconfig.json inherit from this?

Reply:
Sure